packages/webpack/src/presets/base.ts

```diff
@@ -101,16 +100,18 @@ function basePlugins (ctx: WebpackConfigContext) {
               ctx.nuxt.callHook(`${builder}:change`, shortPath)
             }
           },
-          done: ({ state }) => {
-            if (state.hasErrors) {
+          done: (_, { stats }) => {
+            if (stats.hasErrors()) {
               ctx.nuxt.callHook(`${builder}:error`)
             } else {
-              logger.success(`${state.name} ${state.message}`)
+              // @ts-expect-error TODO
+              logger.success(`${stats.name} ${stats.message}`)
             }
           },
           allDone: () => {
             ctx.nuxt.callHook(`${builder}:done`)
           },
+          // @ts-expect-error TODO: there is NO statesArray on type ProgressPlugin, this error is correct
```

Comment:
in that case we probably need to update the hook signature + pass what we have

Reply:
This has been fixed, the only left is the `stats` one.